app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,7 @@
+class FoodsController < ApplicationController 
+  def index 
```

Comment:
This looks clean. How could we slim this down even more? (see my comment in your facade)